deploy/crds/integreatly.org_grafanadashboards_crd.yaml

```diff
@@ -0,0 +1,99 @@
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: GrafanaDashboardSpec defines the desired state of GrafanaDashboard
+          properties:
+            dashboard:
+              description: GrafanaDashboardSpec defines the desired state of GrafanaDashboard
+              properties:
+                json:
+                  description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                    Important: Run "operator-sdk generate k8s" to regenerate code
+                    after modifying this file'
+                  type: string
+                name:
+                  type: string
+                plugins:
+                  items:
+                    description: GrafanaPlugin contains information about a single
+                      plugin
+                    properties:
+                      name:
+                        type: string
+                      version:
+                        type: string
+                    required:
+                    - name
+                    - version
+                    type: object
+                  type: array
+                url:
+                  type: string
+              required:
+              - json
+              - name
+              type: object
+            grafanaRef:
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                Important: Run "operator-sdk generate k8s" to regenerate code after
+                modifying this file'
+              type: string
+          required:
+          - dashboard
+          type: object
+        status:
+          description: GrafanaDashboardStatus defines the observed state of GrafanaDashboard
+          properties:
+            lastConfig:
+              type: string
+            messages:
+              items:
+                properties:
+                  message:
+                    type: string
+                  timestamp:
+                    type: string
+                required:
+                - message
+                - timestamp
+                type: object
+              type: array
+            phase:
+              type: integer
+            slug:
+              type: string
+          required:
+          - phase
+          type: object
+      type: object
+  version: v1alpha1
```

Comment:
@philbrookes @maleck13  Are you familiar with if/how versioning in a CRD can be used to auto migrate CRs when a new version of an operator & CRD are rolled out e.g. from `v1alpha1` to `v1beta1`.